modules/ROOT/pages/tuning-caching.adoc

```diff
@@ -0,0 +1,19 @@
+= How Caching Use Helps to Improve Processing Performance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Identifying key aspects of your data such as when to use cache and which cache strategy to follow, helps you to improve processing performance:
+
+* Is the data frequently called with often repeated results?
+* Can you afford eventual consistency between the data source and the cache information? At least for a while until cache refreshes.
+
+Mule runtime engine (Mule) offers customizable strategies, such as Cache scope and HTTP Caching API Gateway policy, to enable cache according to your needs. +
```

Comment:
```suggestion
Mule runtime engine (Mule) offers customizable strategies, such as the Cache scope and the HTTP Caching API Gateway policy, to enable cache according to your needs. Both of these strategies help applications that constantly request the same data by reducing the processing load, especially network interactions with a remote host.+
```

Reply:
Beware of making each sentence its own separate paragraph. Decide which should be grouped and which not.